packages/rest/src/rest-server.ts

```diff
@@ -47,8 +47,6 @@ interface OpenApiSpecOptions {
 const OPENAPI_SPEC_MAPPING: {[key: string]: OpenApiSpecOptions} = {
   '/openapi.json': {version: '3.0.0', format: 'json'},
   '/openapi.yaml': {version: '3.0.0', format: 'yaml'},
-  '/swagger.json': {version: '2.0', format: 'json'},
-  '/swagger.yaml': {version: '2.0', format: 'yaml'},
```

Comment:
When cleaning up git history, please make sure that the commit message mentions the breaking change - removal of `GET /swagger.{json,yaml}` endpoints.

Reply:
I will add list all breaking changes in the commit message ^